xsd/pmo/options.xsd

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2016-2018 Zerocracy
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to read
+the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+merge, publish, distribute, sublicense, and/or sell copies of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" version="SNAPSHOT" elementFormDefault="qualified" attributeFormDefault="unqualified">
+  <xs:include schemaLocation="http://datum.zerocracy.com/SNAPSHOT/xsd/basics.xsd"/>
+  <xs:complexType name="option">
+    <xs:simpleContent>
+      <xs:extension base="xs:string">
+        <xs:attribute type="xs:string" name="name"/>
+      </xs:extension>
+    </xs:simpleContent>
+  </xs:complexType>
+  <xs:element name="options">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element name="option" type="option" minOccurs="0" maxOccurs="unbounded"/>
+      </xs:sequence>
+      <xs:attribute name="updated" use="required" type="xs:dateTime"/>
+      <xs:attribute name="version" use="required" type="version"/>
+    </xs:complexType>
+    <xs:unique name="uniqueName">
```

Comment:
@g4s8 @yegor256 just wondering out loud here. Maybe we should explicitly define each option as its own type? Having a unique name assumes all options have only one value. This may not necessarily be the case.

Reply:
@yegor256 what do you say?

Reply:
@carlosmiranda I agree with @carlosmiranda Current design of XSD is very generic and will make it impossible for us to validate the XML file properly. Let's define each option here specifically. I would go with this XML:

```
<options>
  <maxJobsInAgenda>10</maxJobsInAgenda>
  <notify>
    <students>false</students>
    <rfps>false</rfps>
  </notify>
  [... etc ...]
</options>
```